chain/client/src/client_actor.rs

```diff
@@ -459,14 +459,18 @@ impl Handler<Status> for ClientActor {
             .map_err(|err| err.to_string())?;
         let latest_block_time = header.inner_lite.timestamp.clone();
         if msg.is_health_check {
-            let elapsed = (Utc::now() - from_timestamp(latest_block_time)).to_std().unwrap();
```

Comment:
`let elapsed = Utc::now().saturating_sub(from_timestamp(latest_block_time))).to_std().unwrap();`?

Reply:
`DateTime<Utc>` does not have `saturating_sub`.